### PR TITLE
Fixed text style in create loading error view

### LINKED ISSUE #10084: Wrong font at "error creating map"
screen shot 2016-10-07 at 11 55 40

cc @MariaCheca

### PR DESCRIPTION
**Pull request by MariaCheca**
Fixes #10084

screen shot 2016-10-13 at 11 48 27

CR @javierarce

### CODE REVIEW

File: lib/assets/javascripts/cartodb/common/views/create/create_loading_error.jst.ejs [hunk 1]
```diff
@@ -16,7 +16,7 @@
       Error creating the new map
     <% } %>
   </h4>
-  <p class="DefaultParagraph DefaultParagraph--short">
+  <p class="CDB-Text CDB-Size-medium u-mainTextColor u-secondaryTextColor">
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/views/create/create_loading_error.jst.ejs]: u-mainTextColor or u-secondaryTextColor?

Comment (MariaCheca) [inline lib/assets/javascripts/cartodb/common/views/create/create_loading_error.jst.ejs]: Oops!

**Comment by MariaCheca** [comment]
@javierarce changes applied!

**Comment by javierarce** [comment]
👍  then

**Pull request merged**